roomserver/version/version.go

```diff
@@ -59,7 +59,7 @@ var roomVersions = map[gomatrixserverlib.RoomVersion]RoomVersionDescription{
 // DefaultRoomVersion contains the room version that will, by
 // default, be used to create new rooms on this server.
 func DefaultRoomVersion() gomatrixserverlib.RoomVersion {
-	return gomatrixserverlib.RoomVersionV2
+	return gomatrixserverlib.RoomVersionV4
```

Comment:
Have you tried this on v3 to see if tests still fail?

Reply:
Yeah - same tests fail, which is to be expected really as the only thing that changes between v3 and v4 is URL-safe encoding of event IDs.